app/api/cruds/utils.py

```diff
@@ -18,3 +18,17 @@ def get_user(relayId, otherwise=None):
         return User.objects.get(pk=from_global_id(relayId)[1])
     except:
         return otherwise
+
+
+def get_object(object_name, relayId, otherwise=None):
+    try:
+        return object_name.objects.get(pk=from_global_id(relayId)[1])
+    except:
+        return otherwise
+
+
+def load_object(instance, args):
+    for key, value in args.items():
+        if getattr(instance, key) and key != "id":
+            setattr(instance, key, value)
+    return instance
```

Comment:
Since you got the idea from here https://github.com/teamtaverna/core/blob/0a9960c77535f7a1326b8085fd88fbb4239ba7da/app/api/cruds/user_crud.py#L79-L83 and abstracted it into a reusable function, I suggest you refactor it to work for all model instances and use it for the UserNode as well. Also, let's find a more suitable name for the function. I suggest `update_object` - what do you think?